ClassroomNotes/src/classroomnotes/LoanCalculator.java

```diff
@@ -4,14 +4,38 @@
  * and open the template in the editor.
  */
 package classroomnotes;
+package classroomnotes;
```

Comment:
the problem that appear is because you duplicate this line, if you remove one of the package definition the problem will be solved